network/p2p/libp2pUtils.go

```diff
@@ -98,37 +101,99 @@ func filterStream(host host.Host, targetID peer.ID, protocol core.ProtocolID, di
 //    |-- Address          --->   |-- []multiaddr.Multiaddr
 //    |-- NetworkPublicKey --->   |-- ID
 func PeerInfoFromID(id flow.Identity) (peer.AddrInfo, error) {
-
-	nodeAddress, err := NodeAddressFromIdentity(id)
-	if err != nil {
-		return peer.AddrInfo{}, fmt.Errorf("failed to convert flow Identity %s to peer.AddrInfo: %w", id.String(), err)
-	}
-
-	addr, err := GetPeerInfo(nodeAddress)
+	addr, err := PeerAddressInfo(id)
```

Comment:
we now just need one of these two functions `PeerInforFromID` and `PeerAddressInfo` since they both do the exact same thing. Hence, you can either remove this or the other.

Reply:
Nice catch 👍  fixed in https://github.com/onflow/flow-go/pull/211/commits/ea1598a5c3cffa8d521f467aab03562db2da8c5e